### ISSUE TITLE
UI idea: Multisig disaster recovery

### ISSUE THREAD

**Issue by relativisticelectron** [cryptoadvance/specter-desktop#1860]
The following scenario:  After a disaster all that remains of the 2/3 multisignature wallet is:

1. seed 1
2. seed 2
3. xpub 3

There seems to be no intuitive way in Specter to recover.  One has to know that one can create a custom device in Specter that contains only xpubs and is not from a signing device.

**Idea**
One could add a button (see below) to make clear one can add the public keys without a physical device:
![image](https://user-images.githubusercontent.com/60378539/186897301-133dc11b-01a4-4e6d-b4ff-50a46e0eb279.png)

**Comment by k9ert** [comment]
I'm not sure how common such a scenario would be. If you have created the wallet with specter, you definitely (should) have a "recovery-json". At least it's MUCH more reasonable to think that you have a "recovery-json" (most probably in the form of a QR-code) then you have the xpub (+2 seeds).
So why make such a recovery available via core? Fine for an extension, though. But not sure we should ship that by default ... ?!

**Comment by relativisticelectron** [comment]
Maybe a disaster recovery manual  in https://docs.specter.solutions/desktop/ would be sufficient?

**Comment by k9ert** [comment]
>  Maybe a disaster recovery manual in https://docs.specter.solutions/desktop/ would be sufficient?

Yes, i think so. There is a podcast in germany called "Nodesignal". They want to sponsor documentation of Specter by doing some sort of documentation bounty. Maybe we should think about what the most urgent missing documentation topics are.
